apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -282,6 +285,43 @@ type AzureKeyVaultCloudInfo struct {
 	ActiveDirectoryEndpoint string `json:"activeDirectoryEndpoint"`
 }
 
+// AwsSecretManager is used to authenticate using AwsSecretManager
+type AwsSecretManager struct {
+	Credentials *AwsSecretManagerCredentials `json:"credentials"`
```

Comment:
I think that this is optional. I mean, do I need `credentials` when using `podIdentity`?